### ISSUE TITLE
Cannot connect to Equinor Volve dataset

### ISSUE THREAD

**Issue by Philliec459** [microsoft/AzureStorageExplorer#3128]
I am trying to connect to the Equinor Volve data set per the link below.

REDACTED URL AND SAS

This is the error message that I am supposed to report. 

```
{
  "name": "TypeError",
  "message": "Cannot read property 'uids' of undefined",
  "__zone_symbol__error": {
    "originalStack": "Error: Cannot read property 'uids' of undefined\n    at AppInsightsAsyncCorrelatedErrorWrapper.ZoneAwareError (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/zone.js/dist/zone-node.js:811:33)\n    at new AppInsightsAsyncCorrelatedErrorWrapper (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/applicationinsights/out/AutoCollection/CorrelationContextManager.js:193:18)\n    at Function.ExceptionSerializer.deserialize (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/se-exception-serializer/dist/ExceptionSerializer.js:49:28)\n    at CloudExplorerProxyMarshaler._finalizeOperation (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/out/app/renderer/marshalers/CloudExplorerProxyMarshaler.js:83:91)\n    at Object.finalizeOperation (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/out/app/renderer/marshalers/CloudExplorerProxyMarshaler.js:107:80)\n    at Function.<anonymous> (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/out/app/renderer/marshalers/StandardMarshaler.js:28:63)\n    at step (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/tslib/tslib.js:141:27)\n    at Object.next (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/tslib/tslib.js:122:57)\n    at /Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/tslib/tslib.js:115:75\n    at new ZoneAwarePromise (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/zone.js/dist/zone-node.js:551:29)\n    at Object.__awaiter (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/tslib/tslib.js:111:16)\n    at JsonPortMarshaler.standardMarshaler.(anonymous function) [as finalizeOperation] (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/out/app/renderer/marshalers/StandardMarshaler.js:22:32)\n    at JsonPortMarshaler.onMessage (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/daytona/pluginHost/messaging/marshaler.js:101:41)\n    at Port.EventTarget.dispatchEvent (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/daytona/pluginHost/common/event.js:69:30)\n    at Port.processMessage (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/daytona/pluginHost/messaging/communication.js:163:18)",
    "zoneAwareStack": "Error: Cannot read property 'uids' of undefined\n    at AppInsightsAsyncCorrelatedErrorWrapper.ZoneAwareError (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/zone.js/dist/zone-node.js:811:33)\n    at new AppInsightsAsyncCorrelatedErrorWrapper (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/applicationinsights/out/AutoCollection/CorrelationContextManager.js:193:18)\n    at Function.ExceptionSerializer.deserialize (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/se-exception-serializer/dist/ExceptionSerializer.js:49:28)\n    at CloudExplorerProxyMarshaler._finalizeOperation (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/out/app/renderer/marshalers/CloudExplorerProxyMarshaler.js:83:91)\n    at Object.finalizeOperation (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/out/app/renderer/marshalers/CloudExplorerProxyMarshaler.js:107:80)\n    at Function.<anonymous> (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/out/app/renderer/marshalers/StandardMarshaler.js:28:63)\n    at step (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/tslib/tslib.js:141:27)\n    at Object.next (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/tslib/tslib.js:122:57)\n    at /Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/tslib/tslib.js:115:75\n    at new ZoneAwarePromise (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/zone.js/dist/zone-node.js:551:29)\n    at Object.__awaiter (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/tslib/tslib.js:111:16)\n    at JsonPortMarshaler.standardMarshaler.(anonymous function) [as finalizeOperation] (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/out/app/renderer/marshalers/StandardMarshaler.js:22:32)\n    at JsonPortMarshaler.onMessage (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/daytona/pluginHost/messaging/marshaler.js:101:41)\n    at Port.EventTarget.dispatchEvent (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/daytona/pluginHost/common/event.js:69:30)\n    at Port.processMessage (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/daytona/pluginHost/messaging/communication.js:163:18)"
  },
  "stack": "TypeError: Cannot read property 'uids' of undefined\n    at StorageExplorerProvider.<anonymous> (file:///Applications/Microsoft%20Azure%20Storage%20Explorer.app/Contents/Resources/app/out/CloudHub/js/Providers/StorageExplorer/StorageExplorerProvider.js:224:66)\n    at step (file:///Applications/Microsoft%20Azure%20Storage%20Explorer.app/Contents/Resources/app/out/CloudHub/js/Providers/StorageExplorer/StorageExplorerProvider.js:47:23)\n    at Object.next (file:///Applications/Microsoft%20Azure%20Storage%20Explorer.app/Contents/Resources/app/out/CloudHub/js/Providers/StorageExplorer/StorageExplorerProvider.js:28:53)\n    at fulfilled (file:///Applications/Microsoft%20Azure%20Storage%20Explorer.app/Contents/Resources/app/out/CloudHub/js/Providers/StorageExplorer/StorageExplorerProvider.js:19:58)",
  "originalStack": "Error: Cannot read property 'uids' of undefined\n    at AppInsightsAsyncCorrelatedErrorWrapper.ZoneAwareError (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/zone.js/dist/zone-node.js:811:33)\n    at new AppInsightsAsyncCorrelatedErrorWrapper (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/applicationinsights/out/AutoCollection/CorrelationContextManager.js:193:18)\n    at Function.ExceptionSerializer.deserialize (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/se-exception-serializer/dist/ExceptionSerializer.js:49:28)\n    at CloudExplorerProxyMarshaler._finalizeOperation (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/out/app/renderer/marshalers/CloudExplorerProxyMarshaler.js:83:91)\n    at Object.finalizeOperation (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/out/app/renderer/marshalers/CloudExplorerProxyMarshaler.js:107:80)\n    at Function.<anonymous> (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/out/app/renderer/marshalers/StandardMarshaler.js:28:63)\n    at step (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/tslib/tslib.js:141:27)\n    at Object.next (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/tslib/tslib.js:122:57)\n    at /Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/tslib/tslib.js:115:75\n    at new ZoneAwarePromise (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/zone.js/dist/zone-node.js:551:29)\n    at Object.__awaiter (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/tslib/tslib.js:111:16)\n    at JsonPortMarshaler.standardMarshaler.(anonymous function) [as finalizeOperation] (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/out/app/renderer/marshalers/StandardMarshaler.js:22:32)\n    at JsonPortMarshaler.onMessage (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/daytona/pluginHost/messaging/marshaler.js:101:41)\n    at Port.EventTarget.dispatchEvent (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/daytona/pluginHost/common/event.js:69:30)\n    at Port.processMessage (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/daytona/pluginHost/messaging/communication.js:163:18)",
  "zoneAwareStack": "Error: Cannot read property 'uids' of undefined\n    at AppInsightsAsyncCorrelatedErrorWrapper.ZoneAwareError (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/zone.js/dist/zone-node.js:811:33)\n    at new AppInsightsAsyncCorrelatedErrorWrapper (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/applicationinsights/out/AutoCollection/CorrelationContextManager.js:193:18)\n    at Function.ExceptionSerializer.deserialize (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/se-exception-serializer/dist/ExceptionSerializer.js:49:28)\n    at CloudExplorerProxyMarshaler._finalizeOperation (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/out/app/renderer/marshalers/CloudExplorerProxyMarshaler.js:83:91)\n    at Object.finalizeOperation (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/out/app/renderer/marshalers/CloudExplorerProxyMarshaler.js:107:80)\n    at Function.<anonymous> (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/out/app/renderer/marshalers/StandardMarshaler.js:28:63)\n    at step (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/tslib/tslib.js:141:27)\n    at Object.next (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/tslib/tslib.js:122:57)\n    at /Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/tslib/tslib.js:115:75\n    at new ZoneAwarePromise (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/zone.js/dist/zone-node.js:551:29)\n    at Object.__awaiter (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/tslib/tslib.js:111:16)\n    at JsonPortMarshaler.standardMarshaler.(anonymous function) [as finalizeOperation] (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/out/app/renderer/marshalers/StandardMarshaler.js:22:32)\n    at JsonPortMarshaler.onMessage (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/daytona/pluginHost/messaging/marshaler.js:101:41)\n    at Port.EventTarget.dispatchEvent (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/daytona/pluginHost/common/event.js:69:30)\n    at Port.processMessage (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/daytona/pluginHost/messaging/communication.js:163:18)",
  "__zone_symbol__stack": "TypeError: Cannot read property 'uids' of undefined\n    at StorageExplorerProvider.<anonymous> (file:///Applications/Microsoft%20Azure%20Storage%20Explorer.app/Contents/Resources/app/out/CloudHub/js/Providers/StorageExplorer/StorageExplorerProvider.js:224:66)\n    at step (file:///Applications/Microsoft%20Azure%20Storage%20Explorer.app/Contents/Resources/app/out/CloudHub/js/Providers/StorageExplorer/StorageExplorerProvider.js:47:23)\n    at Object.next (file:///Applications/Microsoft%20Azure%20Storage%20Explorer.app/Contents/Resources/app/out/CloudHub/js/Providers/StorageExplorer/StorageExplorerProvider.js:28:53)\n    at fulfilled (file:///Applications/Microsoft%20Azure%20Storage%20Explorer.app/Contents/Resources/app/out/CloudHub/js/Providers/StorageExplorer/StorageExplorerProvider.js:19:58)",
  "__zone_symbol__name": "TypeError",
  "__zone_symbol__message": "Cannot read property 'uids' of undefined",
  "__zone_symbol__originalStack": "Error: Cannot read property 'uids' of undefined\n    at AppInsightsAsyncCorrelatedErrorWrapper.ZoneAwareError (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/zone.js/dist/zone-node.js:811:33)\n    at new AppInsightsAsyncCorrelatedErrorWrapper (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/applicationinsights/out/AutoCollection/CorrelationContextManager.js:193:18)\n    at Function.ExceptionSerializer.deserialize (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/se-exception-serializer/dist/ExceptionSerializer.js:49:28)\n    at CloudExplorerProxyMarshaler._finalizeOperation (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/out/app/renderer/marshalers/CloudExplorerProxyMarshaler.js:83:91)\n    at Object.finalizeOperation (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/out/app/renderer/marshalers/CloudExplorerProxyMarshaler.js:107:80)\n    at Function.<anonymous> (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/out/app/renderer/marshalers/StandardMarshaler.js:28:63)\n    at step (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/tslib/tslib.js:141:27)\n    at Object.next (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/tslib/tslib.js:122:57)\n    at /Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/tslib/tslib.js:115:75\n    at new ZoneAwarePromise (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/zone.js/dist/zone-node.js:551:29)\n    at Object.__awaiter (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/tslib/tslib.js:111:16)\n    at JsonPortMarshaler.standardMarshaler.(anonymous function) [as finalizeOperation] (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/out/app/renderer/marshalers/StandardMarshaler.js:22:32)\n    at JsonPortMarshaler.onMessage (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/daytona/pluginHost/messaging/marshaler.js:101:41)\n    at Port.EventTarget.dispatchEvent (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/daytona/pluginHost/common/event.js:69:30)\n    at Port.processMessage (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/daytona/pluginHost/messaging/communication.js:163:18)",
  "__zone_symbol__zoneAwareStack": "Error: Cannot read property 'uids' of undefined\n    at AppInsightsAsyncCorrelatedErrorWrapper.ZoneAwareError (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/zone.js/dist/zone-node.js:811:33)\n    at new AppInsightsAsyncCorrelatedErrorWrapper (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/applicationinsights/out/AutoCollection/CorrelationContextManager.js:193:18)\n    at Function.ExceptionSerializer.deserialize (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/se-exception-serializer/dist/ExceptionSerializer.js:49:28)\n    at CloudExplorerProxyMarshaler._finalizeOperation (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/out/app/renderer/marshalers/CloudExplorerProxyMarshaler.js:83:91)\n    at Object.finalizeOperation (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/out/app/renderer/marshalers/CloudExplorerProxyMarshaler.js:107:80)\n    at Function.<anonymous> (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/out/app/renderer/marshalers/StandardMarshaler.js:28:63)\n    at step (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/tslib/tslib.js:141:27)\n    at Object.next (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/tslib/tslib.js:122:57)\n    at /Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/tslib/tslib.js:115:75\n    at new ZoneAwarePromise (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/zone.js/dist/zone-node.js:551:29)\n    at Object.__awaiter (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/tslib/tslib.js:111:16)\n    at JsonPortMarshaler.standardMarshaler.(anonymous function) [as finalizeOperation] (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/out/app/renderer/marshalers/StandardMarshaler.js:22:32)\n    at JsonPortMarshaler.onMessage (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/daytona/pluginHost/messaging/marshaler.js:101:41)\n    at Port.EventTarget.dispatchEvent (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/daytona/pluginHost/common/event.js:69:30)\n    at Port.processMessage (/Applications/Microsoft Azure Storage Explorer.app/Contents/Resources/app/node_modules/daytona/pluginHost/messaging/communication.js:163:18)"
}
```

**Comment by MRayermannMSFT** [comment]
@philliec459 what is your operating system?

**Comment by Philliec459** [comment]
Hi,

I am working on a macbook pro.



Best Regards, 
 
_______________________________
 
E. Craig Phillips
CEO and Chief Petrophysicist
Crested Butte Petrophysical Consultants
 







459 Cisneros Lane
Crested Butte, CO 81224
USA
Office: +1 970-343-0730
Mobile: +1 970-343-0730
email: craig@cbpetro.com
Website: www.cbpetro.com
GitHub: https://github.com/Philliec459



The information contained in this electronic message is confidential, it constitutes a profesional and/or industrial secret in terms of the current legislation, and is intended for its recipient only. If you receive this message by mistake or if you are not the recipient thereof, please notify the sender and destroy it.

La informacion contenida en este mensaje de datos es confidencial, constituye un secreto industrial y/o profesional en terminos de la legislacion vigente y se encuentra dirigida exclusivamente al destinatario indicado en dicho mensaje. Si usted recibe esta informacion por error o si usted no es el destinatario del mensaje, favor de notificar al emisor, y destruyalo.

On Jun 22, 2020, at 11:37 AM, Matthew Rayermann <notifications@github.com> wrote:


@Philliec459 <https://github.com/Philliec459> what is your operating system?

**Comment by Philliec459** [comment]
Is there any update on the problem?

**Comment by JasonYeMSFT** [comment]
@Philliec459 Usually this type of issue is caused by corrupted data in the browser local storage. Unfortunately we haven't been able to figure out why the local storage data is corrupted on some users' machines but we have seen the "Error occurred while adding new connection: TypeError: Cannot read property 'version' of undefined" steps in our troubleshooting guide helpful in working around this issue [troubleshooting guide](https://docs.microsoft.com/en-us/azure/storage/common/storage-explorer-troubleshooting?tabs=Windows%2C2004#error-occurred-while-adding-new-connection-typeerror-cannot-read-property-version-of-undefined).

In short, the instruction tells you how to clear the existing data stored for that connection so that you can re-add that connection. Please try the steps to see if helps and let us know if the issue still reproduces.

Thanks

**Comment by Philliec459** [comment]
Thank you,

I have no real data to clear. However, since I have tried this a few times already, there is probably some file that is hindering me now. I will clear everything and start over again. 


Best Regards, 
 
_______________________________
 
E. Craig Phillips
CEO and Chief Petrophysicist
Crested Butte Petrophysical Consultants
 







459 Cisneros Lane
Crested Butte, CO 81224
USA
Office: +1 970-343-0730
Mobile: +1 970-343-0730
email: craig@cbpetro.com
Website: www.cbpetro.com
GitHub: https://github.com/Philliec459



The information contained in this electronic message is confidential, it constitutes a profesional and/or industrial secret in terms of the current legislation, and is intended for its recipient only. If you receive this message by mistake or if you are not the recipient thereof, please notify the sender and destroy it.

La informacion contenida en este mensaje de datos es confidencial, constituye un secreto industrial y/o profesional en terminos de la legislacion vigente y se encuentra dirigida exclusivamente al destinatario indicado en dicho mensaje. Si usted recibe esta informacion por error o si usted no es el destinatario del mensaje, favor de notificar al emisor, y destruyalo.

On Jun 29, 2020, at 2:57 PM, JasonYeMSFT <notifications@github.com> wrote:


@Philliec459 <https://github.com/Philliec459> Usually this type of issue is caused by corrupted data in the browser local storage. Unfortunately we haven't been able to figure out why the local storage data is corrupted on some users' machines but we have seen the "Error occurred while adding new connection: TypeError: Cannot read property 'version' of undefined" steps in our troubleshooting guide helpful in working around this issue troubleshooting guide <https://docs.microsoft.com/en-us/azure/storage/common/storage-explorer-troubleshooting?tabs=Windows%2C2004#error-occurred-while-adding-new-connection-typeerror-cannot-read-property-version-of-undefined>.

In short, the instruction tells you how to clear the existing data stored for that connection so that you can re-add that connection. Please try the steps to see if helps and let us know if the issue still reproduces.

Thanks

**Comment by JasonYeMSFT** [comment]
@Philliec459, if the troubleshooting step doesn't work, it would be great if we can schedule a call to debug the issue on your machine so that we can help you resolve the issue as well as study the actual cause of it. Let us know if it is possible, what time do you prefer and what kind of tool (e.g. Teams, Skype, Slack, Zoom etc) do you prefer to make the call.

Thanks

**Comment by MRayermannMSFT** [comment]
Closing due to lack of response. If you are still having issues, please open up a new issue, thanks.
